app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity
+            android:name=".MainActivity"
+            android:configChanges="orientation|screenSize">
```

Comment:
Please don't lock the orientation. You don't kill the patient when he has a disease

Reply:
Actually this does not lock the orientation. It is just that that activity does not get re created on configuration changes.

Reply:
location is not locked!

Reply:
Which means that the orientation of layout is locked. If you implement tablet layout in future, it won't be applied on rotation. This is not correct behaviour since we are already using ViewModel and other lifecycle aware components

Reply:
Even the screen size is locked which will create problems with Android Multi Tasking view and PiP